### ISSUE TITLE
Can get asset field/file input to work in update form

### ISSUE THREAD

**Issue by hmalaud** [jacksleight/statamic-memberbox#5]
## Bug Description
When trying to update profile with asset field, form throws an error considering this field as an array 
`The avatar must be an array.
The avatar may not have more than 1 items.`

Couldn't find any mention of asset fields in the doc, might that not be covered by the addon?

## How to Reproduce
1. Add asset type field to user profile `resources/blueprints/user.yaml`
```
-
  handle: avatar
  field:
    type: assets
    max_files: 1
    instructions_position: above
    listable: hidden
    mode: list
    container: assets
    folder: avatars
    restrict: true
    allow_uploads: true
    show_filename: true
```
2. Publish form files `php please vendor:publish --tag=statamic-memberbox-views`

3. Add `files="true"` on update profile form `resources/views/vendor/statamic-memberbox/web/profile.antlers.html`
```{{ mb:user:profile_form files="true" }}```

4. Add file input field to form
```
 <div class="mb-5">
   <label for="avatar" class="block mb-1">Avatar</label>
   <input class="bg-white p-2 w-full rounded-md" type="file" name="avatar" id="avatar" />
</div>
```

5. Submit form.

## Environment
Statamic 3.2.22 Pro
Laravel 8.70.2
PHP 8.0.12
jacksleight/statamic-memberbox 0.4.2
statamic/ssg 0.8.0
stillat/meerkat 2.4.9-beta3

**Comment by jacksleight** [comment]
Ah yeah, not currently supported, but should be. Will take a look.

**Comment by jacksleight** [comment]
Thanks for the report, this is now fixed in v0.4.3.

Docs have been updated: https://jacksleight.github.io/statamic-memberbox/customisation.html#file-uploads

FYI, there's currently a bug in Statamic core that means asset fields used for frontend file uploads must have a container set, if it's not set you'll get an error.

**Comment by hmalaud** [comment]
Well, that was fast, thanks!
